content/common/linux.mk

```diff
@@ -0,0 +1,12 @@
+CC := gcc
+CFLAGS ?= -g -Wall -Wextra -Werror -I ../utils
```

Comment:
I think it would be difficult to use `../utils`. It would be relative to the current source code folder, but that means having a `utils/` folder in the parent of each folder. Ideally we would place it also here in the `common/` folder. Meaning, it would be relative to the Makefile folder; so you would use a construct such as:

```
COMMON_DIR := $(dir $(lastword $(MAKEFILE_LIST)))
CPPFLAGS += -I $(COMMON_DIR)/../utils
```